packages/paste-website/src/component-examples/InPageNavigationExamples.ts

```diff
@@ -0,0 +1,45 @@
+export const basicExample = `
+const BasicInPageNavigation = () => {
+
+  return (
+    <InPageNavigation aria-label="my-nav">
```

Comment:
Issue(blocking): aria label has to be a human readable string. So it could be "Primary navigation" or something like that. Same with the full width example.

Reply:
the stories should also be updated to human readable strings